### ISSUE TITLE
[BUG] Unable to detect Multiple faces in Video

### ISSUE THREAD

**Issue by aarpit1010** [aarpit1010/Real-Time-Face-Mask-Detector#15]
When the `detection.ipynb` is run for a random video file instead of a webcam then  it was only able to detect one face at a time.
Edit the the code by adding or removing few lines of code and see whether it is able to detect multiple faces for the video provided or not.

HINT : predict function is expecting an np array and instead of that we were serving it a list.
           To fix this you need to remove `face = np.expand_dims(face, axis=0)` and replace it with some other line so that multiple 
           faces can be detected.

Multiple people can work on the issue at a time.
Only the 1st correct PR would be merged.

**Comment by Mridul20** [comment]
i want to work on this issue.

**Comment by aarpit1010** [comment]
@Mridul20  you can work on this but make sure that the dimensions don't get disturbed.
You can also make the use of another pre-trained model that you can built with no-code from [this website](https://teachablemachine.withgoogle.com/)

**Comment by Mridul20** [comment]
![Untitled](https://user-images.githubusercontent.com/53915334/96549014-817ccc00-12cc-11eb-8660-f6bd9829f676.png)

@aarpit1010  Please verify.

**Comment by aarpit1010** [comment]
ok, good work ...
but there is still room for improvements...
do try out the `keras_model.h5` that I have uploaded recently, also try out implementing the code on a video having people wearing masks and some people with no masks..
You can ask whatever doubts you have here or on discord server
